ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+## Current state and roadmap
+
+The following is a loose collection of potential features that fit the future
+scope of the operator. Their exact implementation and viability are subject
+to further discussion.
+
+### Rule files
+
+The Operator creates an empty ConfigMap of the name `<prometheus-name>-rules` if it
+doesn't exist yet. It is left to the user to populate it with the desired rules.
+
+It is still up for discussion whether it should be possible to include rule files living
+in arbitrary ConfigMaps by their labels.
+Intuitively, it seems fitting to define in each `ServiceMonitor` which rule files (based 
+label selections over ConfigMaps) should be deployed with it.
+However, rules act upon all metrics in a Prometheus server. Hence, defining the
+relationship in each `ServiceMonitor` may cause undesired interference.
+ 
+### Alerting
+
+The TPR allows to configure multiple namespace/name pairs of Alertmanagers
+services. The Prometheus instances will send their alerts to each endpoint
+of this service.
+
+Currently Prometheus only allows to configure Alertmanager URLs via flags
+on startup. Thus the Prometheus pods have to be restarted manually if the 
+endpoints change.
+PetSets or manually maintained headless services in Kubernetes allow to
+provide stable URLs working around this. In the future, Prometheus will allow
+for dynamic service discovery of Alertmanagers ([tracking issue](https://github.com/prometheus/prometheus/issues/2057)). 
+
+### Cluster-wide version
```

Comment:
this is outdated, and will only be updated by external systems and the operator performs updates based on changes to single Prometheus objects